locations/spiders/remax_de.py

```
@@ -1,41 +1,12 @@
-                        open_time=days.get("opens"),
-                        close_time=days.get("closes"),
-                    )
-            item["opening_hours"] = oh
-
-            yield item
-
-        page = int(re.findall(r"\d+$", response.url)[0]) + 1
-        if len(response.json()):
-            url = f"https://wp.ooremax.com/wp-json/eapi/v1/agencies?per_page={self.per_page}&page={page}"
-            yield scrapy.Request(url=url, callback=self.parse)
+    start_urls = ["https://www.remax.de/en/real-estate-offices-agents"]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)